src/util/error.js

```diff
@@ -4,10 +4,19 @@
  * @author Box
  */
 
+import messages from '../components/messages';
+
 export function getBadItemError(): Error {
     return new Error('Bad box item!');
 }
 
 export function getBadPermissionsError(): Error {
     return new Error('Insufficient Permissions!');
 }
+
+export function getActivityFeedApiError(): InlineError {
```

Comment:
this feels specific to the AF, not a general util that will be used elsewhere. Can we move to ActivityFeed.js?